modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -319,21 +319,24 @@ export class {{classname}} {
 
 {{/hasFormParams}}
 {{#useHttpClient}}
+        const httpOptions: Object = {
+            {{#hasQueryParams}}
+            params: queryParameters,
+            {{/hasQueryParams}}
+            withCredentials: this.configuration.withCredentials,
+            headers: headers,
+            observe: observe,
+            reportProgress: reportProgress,
+            {{^isResponseFile}}
+            responseType: this.configuration.selectResponseType(httpHeaderAcceptSelected)
```

Comment:
I think the type needs to be set explicitly, can we determine the response type when generating the api?

Reply:
yes you're right , this change doesn't work